documentation/IDTA-01001/modules/ROOT/IDTA-01001.adoc

```diff
@@ -0,0 +1,7498 @@
+////
+Copyright (c) 2023 Industrial Digital Twin Association
+
+This work is licensed under a [Creative Commons Attribution 4.0 International License](
+https://creativecommons.org/licenses/by/4.0/). 
+
+SPDX-License-Identifier: CC-BY-4.0
+
+Illustrations:
+Plattform Industrie 4.0; Anna Salari, Publik. Agentur für Kommunikation GmbH, designed by Publik. Agentur für Kommunikation GmbH
+////
+
+:toc: left
+:toc-title: Specification of the Asset Administration Shell. Part 1: Metamodel
+:sectlinks:
+:sectnums:
+:imagesdir: images/
+:nofooter:
+
+
+include::../Constraints/constraints.adoc[]
+
+
+= image:img.png[width=100%]
```

Comment:
Do we want to keep the current cover sheet image or should we replace it with a text?